charts/sn-platform/templates/tls/tls-certs-internal.yaml

```diff
@@ -103,8 +103,12 @@ spec:
     - client auth
   # At least one of a DNS Name, USI SAN, or IP address is required.
   dnsNames:
+  {{- if and .Values.istio.enabled (eq .Values.istio.gateway.tls.mode "PASSTHROUGH") }}
```

Comment:
Is the cert used for internal tls connection only?

Reply:
yeah, I think even without Istio, user should still able to generate and use internal cert for mTls, this condition could be a breaking change.
I think we should generate a new cert for this purpose if needed

Reply:
Internal TLS shouldn't work with Istio Gateway PASSTHROUGH mode. If Istio is disabled, internal TLS mode will not be affected. But maybe defining a new cert for this is better.